### ISSUE TITLE
Reinforced alloy

### ISSUE THREAD

**Issue by cigue1978** [SacrificialFlame/Beyond-The-Void-2-Reborn#8]
*Please check the open and  closed issues before posting an issue!*

#### Issue description:

Is the recipe for reinforced alloy really intended to use 16 reinforced dust? Considering that one reinforced dust uses one diamond dust (so one diamond) that is one pricey recipe. Or is there another way to get to flying (to be able to navigate the nether - without the cheaty fly command) that I not see other than using the jetpack from mekanica which I need the pressurized reaction chamber for. But the pressurized reaction chamber needs tungsten steel which I only can get (without the nether chickens - for which I need flying) by building the blast furnace multiblock which in turn needs advanced machine casing (34) which each needs two elite control circuits which needs two reinforced alloy. So 136 reinforced alloy times 16 reinforced dust (times on diamond) - that makes it 2.176 diamonds for the blast furnace to get those damn 4 tungsten steel plates for the pressurized reaction chamber to get the hydrogen as fuel for my jetpack. I understand that this is an expert pack but so many diamonds for one recipe? I feel it is a little bit unbalanced.

#### Steps to reproduce:
1.
2.
3.

#### Version (make sure you are on the latest version before reporting):
**Forge:** 14.23.5.2808
**Mekanism:** 
**Other relevant version:** actual pack version 2.7.0

#### If a (crash)log is relevant for this issue, link it here: (_It's almost always relevant_)

[gist/pastebin/etc link here]

**Comment by SacrificialFlame** [comment]
The only Reinforced Alloy Recipe I see in JEI is Mekanica's and that does not show anything your talking about

**Comment by cigue1978** [comment]
I do mean mekanica's recipe - and I just realized that I totally had the name of the dust wrong, sorry for that - it is refined obsidian dust I meant - and for creating one reinforced alloy I need 160 units(?) of refined obsidian dust - which correspondends to 16 pieces of refined obsidian dust. One refined obsidian dust needs one diamond. I just wanted to know if this was intended - because it still scales up to more than 2000 diamonds if this unit of refined obsidian dust is intended for the recipe.

**Comment by SacrificialFlame** [comment]
Weird cause I see the recipe and it shows nothing with Diamonds so kinda confused

**Comment by SacrificialFlame** [comment]
Think I fixed it or at least added another way

**Comment by cigue1978** [comment]
I just saw in the recipe in the metallurgic infuser that the left column (where the infusing item goes into (like redstone and such - and is stored there - opposite the slot where the stored energy is shown) it says only "**obsidian: 160**" while the needed infusing item is shown as **refined obsidian dust** which then is infused with enriched alloy (made from redstone (160 units) and iron) to create the reinforced alloy. Maybe that is the problem here. Two different names.

Just to clarify: all six slots of the recipe to me are shown as follows (from left to right):  1) obsidian 160, 2) refined obsidian dust, 3) enriched alloy --> 4) reinforced alloy, 5) slot with energy symbol, 6) slot showing stored energy (green).

And the refined obsidian dust (as another recipe) is made in the metallurgic infuser from diamond dust and pulverized obsidian (I used the thermal recipe for that).
